src/pytorch_metric_learning/losses/cross_batch_memory.py

```diff
@@ -8,6 +9,10 @@
 class CrossBatchMemory(ModuleWithRecords):
     def __init__(self, loss, embedding_size, memory_size=1024, miner=None, **kwargs):
         super().__init__(**kwargs)
+        warnings.warn(
```

Comment:
Is there a way to include this warning without code duplication? Maybe this could work:

```python
from ..wrappers import CrossBatchMemory as NewCrossBatchMemory

def CrossBatchMemory(*args, **kwargs):
     warnings.warn(
            "Importing MultipleLosses from losses is deprecated. Import MultipleLossesWrapper from pytorch_metric_learning.wrappers instead.",
            DeprecationWarning
        )
    return NewCrossBatchMemory(*args, **kwargs)
```

If not, then I'd prefer deleting these files to avoid duplicate code. The warnings are nice, but if these changes are introduced as "v2.0" then I think breaking changes are ok.

Reply:
It's possible this might introduce a circular import error though.